src/Uno.UI.Toolkit/ElevatedView.cs

```diff
@@ -78,9 +78,9 @@ public Color ShadowColor
 		}
 
 		public static readonly DependencyProperty ContentProperty = DependencyProperty.Register(
```

Comment:
```suggestion
		public static readonly DependencyProperty ElevatedContentProperty = DependencyProperty.Register(
```

The other names need to be aligned

Reply:
![](https://media.giphy.com/media/EX6wKXVM9IOlO/giphy.gif)